### ISSUE TITLE
Joshua please fix the official pcie Realtek RTL8852BE-CG

### ISSUE THREAD

**Issue by InfernalWolf666** [Joshua-Riek/ubuntu-rockchip#428]
official pcie Realtek RTL8852BE-CG 
http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-R6.html

Joshua please fix the bluetooth.
Wi-Fi in version 1.28 works fine, but bluetooth does not work.

I have already tried copying files from official versions of Linux from the orange pi. Does not work!

**Comment by Joshua-Riek** [comment]
This has already been addressed and fixed. Waiting on my fix to get merged, then the wifi and Bluetooth module will work correctly.

**Comment by Joshua-Riek** [comment]
Closing as a duplicate of #422
